config/aaa.py

```diff
@@ -11,6 +12,41 @@
 RADIUS_MAXSERVERS = 8
 RADIUS_PASSKEY_MAX_LEN = 65
 VALID_CHARS_MSG = "Valid chars are ASCII printable except SPACE, '#', and ','"
+TACACS_PASSKEY_MAX_LEN = 65
+TACACS_SECRET_SALT = "2e6593364d369fba925092e0c1c51466c276faa127f20d18cc5ed8ae52bedbcd"
+
+def get_salt():
+    file_path = "/etc/shadow"
+    target_username = "admin"
```

Comment:
this variable not used, can be cleaned up

Reply:
Sure.

Reply:
Addressed.